### ISSUE TITLE
Why you don't use any skip connection?

### ISSUE THREAD

**Issue by DevinXYang** [zsef123/EfficientNets-PyTorch#2]
https://github.com/zsef123/EfficientNets-PyTorch/blob/0a8e18486af1ef24af0b73e1e4e985d457e2ee07/effnet.py#L97

`self.skip` always equal `False`?

**Comment by zsef123** [comment]
My codes is wrong.
`skip` parameters are always be `True`
[TF Builder # 67](https://github.com/mingxingtan/efficientnet/blob/91f3a0dc67fb458b6a06263e96c983394004ec79/efficientnet_builder.py#L67)
[TF Builder # 122 ](https://github.com/mingxingtan/efficientnet/blob/91f3a0dc67fb458b6a06263e96c983394004ec79/efficientnet_builder.py#L122)
And I'm finding bug in dropconnect condition.

Now I'm coding about load `pretrained TF weight` to my model.
So I will push fixed model when the conversion code is complete.

**Comment by zsef123** [comment]
Please check this commit 
https://github.com/zsef123/EfficientNets-PyTorch/commit/1ce28a3fe9394c3c72b1aa0ea00b723747a5e334

**Comment by DevinXYang** [comment]
That’s really good work!

**Comment by DevinXYang** [comment]
@zsef123 Hi, I have some confuse on `SamePadConv2d.`
If I only need 224 input, do I need this?

**Comment by zsef123** [comment]
`SamePadConv2d` does not matter to Input Image size

It made for equivalent with TF Padding algorithm

I'm not sure about difference with TF and Pytorch Padding algorithm.

But In previous code, `nn.Conv2d`, acc of pretrained network is lower than paper figure.
In now, `SamePadConv2d`, acc is very close to figure

**Comment by DevinXYang** [comment]
Thanks that！
